app/services/common_platform/connection.rb

```diff
@@ -24,24 +22,37 @@ def call
         connection.response :json, content_type: "application/json"
         connection.response :json, content_type: "application/vnd.unifiedsearch.query.laa.cases+json"
         connection.response :json, content_type: "text/plain"
-        connection.adapter Faraday.default_adapter
+        connection.adapter :net_http_persistent, {
```

Comment:
this keeps the http connection open for 60 sec, speeding up re response in case of high traffic.